engine/core/src/test/java/org/datacleaner/util/http/CasMonitorHttpClientTest.java

```diff
@@ -30,7 +30,7 @@
 import org.apache.http.client.methods.HttpUriRequest;
 import org.apache.http.impl.client.HttpClients;
 
-public class CASMonitorHttpClientTest {
+public class CasMonitorHttpClientTest {
```

Comment:
Since the name is still CASMonitorHttpClient I think we should also keep that in the test class name.